.storybook/stories/modal/sidepanel.stories.ts

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrSidepanel, ClrSidepanelModule, commonStringsDefault } from '@clr/angular';
+import { action } from '@storybook/addon-actions';
+import { moduleMetadata, StoryFn, StoryObj } from '@storybook/angular';
+
+import { CommonModules, removeFocusOutline } from '../../helpers/common';
+
+export default {
+  title: 'Modal/Side Panel',
+  decorators: [
+    moduleMetadata({
+      imports: [...CommonModules, ClrSidepanelModule],
+    }),
+  ],
+  component: ClrSidepanel,
+  argTypes: {
+    // inputs
+    clrSidepanelCloseButtonAriaLabel: { type: 'string', defaultValue: commonStringsDefault.close },
+    clrSidepanelLabelledById: { defaultValue: '' },
+    clrSidepanelSize: {
+      defaultValue: 'md',
+      options: ['sm', 'md', 'lg', 'xl', 'full-screen'],
+      control: { type: 'radio' },
+    },
+    clrSidepanelSkipAnimation: { defaultValue: false, control: { type: 'boolean' } },
+    // outputs
+    clrSidepanelOpenChange: { control: { disable: true } },
+    clrSidepanelAltClose: { control: { disable: true } },
+    // methods
+    fadeDone: { control: { disable: true }, table: { disable: true } },
+    open: { control: { disable: true }, table: { disable: true } },
+    close: { control: { disable: true }, table: { disable: true } },
+  },
+  args: {
+    // outputs
+    clrSidepanelOpenChange: action('clrSidepanelOpenChange'),
+    clrSidepanelAltClose: action('clrSidepanelAltClose'),
+    // story helpers
+    title: 'Side Panel Title',
+    body: 'Hello World!',
+  },
+};
+
+const SidepanelTemplate: StoryFn = args => ({
```

Comment:
```suggestion
const SidePanelTemplate: StoryFn = args => ({
```

nit: consistent casing

Reply:
Wait, maybe it's the other ones in this file that are wrong. We should pick one word or two and be consistent.

Reply:
Yeah, I was thinking about Sidepanel (after Sidebar) vs SidePanel, but your suggestion may be the better one.
Will change it everlywhere needed.

Reply:
Done. Sidebar is one word, but Side panel should be two words I think. At least design request is like that so I stick with the 2 words.